.gitignore

```diff
@@ -20,4 +20,4 @@ services/database/local_buckets
 tests/test-results/
 tests/playwright-report/
 tests/playwright/.cache/
-tests/playwright/.auth
+tests/.auth/
```

Comment:
I didn't like the `.auth` folder buried in the `playwright` folder. Open to debate.